cmd/openshift/operator/kodata/tekton-addon/addons/02-clustertasks/source_local/buildah/buildah-task.yaml

```diff
@@ -59,6 +59,13 @@ spec:
       The file should be placed at the root of the Workspace with name config.json
       or .dockerconfigjson.
     optional: true
+  - description: >-
+      An optional workspace that allows providing a entitlement keys
+      for Buildah to access subscription. The file should be placed
```

Comment:
nit: the second line is very confusing

Reply:
@piyush-garg updated words 
Let me know if this is still confusing